templates/resources/fabric_connection.md.tmpl

```diff
@@ -94,6 +94,49 @@ EVPLAN Port to Network Connection:
 
 {{tffile "examples/resources/equinix_fabric_connection/fcr_to_metal.tf"}}
 
+Port to Alibaba Connection EVPL_VC Connection:
+
+{{tffile "examples/resources/equinix_fabric_connection/port_to_alibaba.tf"}}
+
+### Step by Step Instructions for Fabric Port to Fabric Alibaba Profile Connection
```

Comment:
To provide even more clarity you could rephrase this to `Step by Step Instructions for the Fabric Port to Fabric Alibaba Profile Connection Example Given Below`

Reply:
Done ✔️